.env.example

```diff
@@ -1,28 +1,41 @@
-BUSINESS_URL = "https://dev.bcregistry.ca/business/auth/"
-COLIN_URL = "https://www.corporateonline.gov.bc.ca"
-CSO_LEARN_MORE_URL = "https://justice.gov.bc.ca/cso/index.do"
-CSO_URL = "https://dev.justice.gov.bc.ca/cso/index.do"
-ESRA_LEARN_MORE_URL = "https://esra-prd-frontend.apps.silver.devops.gov.bc.ca/"
-ESRA_URL = "https://esra-dlvr-frontend.apps.silver.devops.gov.bc.ca/"
-KEYCLOAK_CONFIG_PATH = "keycloak-dev.json"
-NAME_REQUEST_URL = "https://dev.bcregistry.ca/namerequest/"
-ORGBOOK_URL = "https://dev.orgbook.gov.bc.ca/"
-PPR_URL = "https://ppr-ui-dev.apps.silver.devops.gov.bc.ca/ppr"
-REGISTRY_ACCOUNT_CREATE_URL = "https://dev.bcregistry.ca/business/auth/choose-authentication-method"
+#vaults web-url
+ADDRESSCHANGE_URL="https://www.addresschange.gov.bc.ca/"
+BCA_LEARN_MORE_URL="https://assessmentreportsonline-test.bcassessment.ca:54585/"
+BCA_URL="https://bcrosweb-dev.bcassessment.ca:54579/"
+BUS_SEARCH_LEARN_MORE="https://www2.gov.bc.ca/gov/content?id=B75BE1375F084B138B60D62C0094D9E8"
+COLIN_URL="https://www.corporateonline.gov.bc.ca"
+CSO_LEARN_MORE_URL="https://justice.gov.bc.ca/cso/index.do"
+CSO_URL="https://dev.justice.gov.bc.ca/cso/index.do"
+ESRA_LEARN_MORE_URL="https://www2.gov.bc.ca/gov/content/environment/air-land-water/site-remediation/site-information"
+ESRA_URL="https://esra-dlvr-frontend.apps.silver.devops.gov.bc.ca/"
+REGISTRY_ACCOUNT_CREATE_URL="https://dev.bcregistry.ca/business/auth/choose-authentication-method"
 RPT_URL="https://staging.www.etax.gov.bc.ca/bts/Web.BCOnl/WebServices/BCO/login"
-SITEMINDER_LOGOUT_URL = "https://logontest7.gov.bc.ca/clp-cgi/logoff.cgi"
-SOCIETIES_URL = "https://dev.bcregistry.ca/societies/"
-STATUS_API_URL = "https://status-api-dev.apps.silver.devops.gov.bc.ca/api/v1/"
-WILLS_URL = "https://ewills-dev.vs.gov.bc.ca/"
-WILLS_LEARN_MORE_URL = "https://www2.gov.bc.ca/gov/content/life-events/death/wills-registry"
+SOCIETIES_URL="https://dev.bcregistry.ca/societies/"
+WILLS_URL="https://ewills-dev.vs.gov.bc.ca/"
+WILLS_LEARN_MORE_URL="https://www2.gov.bc.ca/gov/content/life-events/death/wills-registry"
+
+AUTH_URL="https://dev.bcregistry.ca/business/auth/"
+BUSINESS_URL="https://dev.bcregistry.ca/business/auth/"
+BCROS_BASE_URL="https://dev.bcros.ca/"
+BUSINESS_SEARCH_URL="https://dev.bcregistry.ca/business/search"
+BCOL_URL="https://d1.bconline.gov.bc.ca/"
+BCEID_URL="https://www.test.bceid.ca/os/?7686&SkipTo=Basic"
+NAME_REQUEST_URL="https://dev.bcregistry.ca/namerequest/"
+ORGBOOK_URL="https://orgbook.gov.bc.ca/"
+PPR_URL="https://ppr-ui-dev.apps.silver.devops.gov.bc.ca/ppr/"
+SITEMINDER_LOGOUT_URL="https://logontest7.gov.bc.ca/clp-cgi/logoff.cgi"
+
+#vaults API
+AUTH_API_URL="https://auth-api-dev.apps.silver.devops.gov.bc.ca"
+AUTH_API_VERSION="/api/v1"
+
+STATUS_API_URL="https://status-api-dev.apps.silver.devops.gov.bc.ca"
+STATUS_API_VERSION="/api/v1"
+
+#vaults launchdarkly
+BCREGISTRY_LD_CLIENT_ID="6334d2fc9cdd111112927a47"
```

Comment:
We should not commit this value. When running locally, the local fallback LD flags should be used (or the developer can fill in this field for themselves).